itstool fails to build on amd64

The package itstool fails to build from source with the following error:

```
using /tmp/apko-temp-270367864/apko-x86_64.tar.gz for image layer
ImgRef = /tmp/melange-guest-1445516496
running step "fetch"
--2024-12-17 07:10:23--  https://github.com/itstool/itstool/archive/2.0.7/itstool-2.0.7.tar.gz
Resolving github.com... 140.82.112.3
Connecting to github.com|140.82.112.3|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://codeload.github.com/itstool/itstool/tar.gz/refs/tags/2.0.7 [following]
--2024-12-17 07:10:23--  https://codeload.github.com/itstool/itstool/tar.gz/refs/tags/2.0.7
Resolving codeload.github.com... 140.82.112.9
Connecting to codeload.github.com|140.82.112.9|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: unspecified [application/x-gzip]
Saving to: 'itstool-2.0.7.tar.gz'

     0K .......... .......... .......... .......... ..........  684K
    50K .......... .......... ........                         61.1M=0.07s

2024-12-17 07:10:23 (1.04 MB/s) - 'itstool-2.0.7.tar.gz' saved [80082]

fetch: Expected sha256: fba78a37dc3535e4686c7f57407b97d03c676e3a57beac5fb2315162b0cc3176
autoreconf: export WARNINGS=
autoreconf: Entering directory '.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal --force 
autoreconf: configure.ac: tracing
autoreconf: configure.ac: not using Libtool
autoreconf: configure.ac: not using Intltool
autoreconf: configure.ac: not using Gtkdoc
autoreconf: running: /usr/bin/autoconf --force
autoreconf: configure.ac: not using Autoheader
autoreconf: running: automake --add-missing --copy --force-missing
configure.ac:2: installing './install-sh'
configure.ac:2: installing './missing'
autoreconf: Leaving directory '.'
checking for a BSD-compatible install... /usr/bin/install -c
checking whether sleep supports fractional seconds... yes
checking filesystem timestamp resolution... 2
checking whether build environment is sane... yes
checking for a race-free mkdir -p... /bin/mkdir -p
checking for gawk... no
checking for mawk... no
checking for nawk... no
checking for awk... awk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking xargs -n works... yes
checking whether /usr/bin/python3 version is >= 2.6... yes
checking for /usr/bin/python3 version... 3.13
checking for /usr/bin/python3 platform... linux
checking for GNU default /usr/bin/python3 prefix... ${prefix}
checking for GNU default /usr/bin/python3 exec_prefix... ${exec_prefix}
checking for /usr/bin/python3 script directory (pythondir)... ${PYTHON_PREFIX}/lib/python3.13/site-packages
checking for /usr/bin/python3 extension module directory (pyexecdir)... ${PYTHON_EXEC_PREFIX}/lib/python3.13/site-packages
checking for python module libxml2... not found
configure: error: Python module libxml2 is needed to run this package
deleting guest dir /tmp/melange-guest-798221235
deleting workspace dir /tmp/melange-workspace-3260105973
removing image path /tmp/melange-guest-1445516496
failed to build package: unable to run package itstool pipeline: unable to run pipeline: exit status 1
```

--- a/itstool.yaml
+++ b/itstool.yaml
@@ -66,6 +66,9 @@ test:
         - gettext
         - python3<3.13
   pipeline:
+    - runs: |
+        itstool --help
+        itstool --version | grep -q "${{package.version}}"
     - uses: fetch
       with:
         expected-sha256: fba78a37dc3535e4686c7f57407b97d03c676e3a57beac5fb2315162b0cc3176

--- a/itstool.yaml
+++ b/itstool.yaml
@@ -56,3 +56,23 @@ update:
     identifier: itstool/itstool
     strip-prefix: v
     use-tag: true
+
+test:
+  environment:
+    contents:
+      packages:
+        - diffutils
+        - docbook-xml
+        - gettext
+        - python3<3.13
+  pipeline:
+    - uses: fetch
+      with:
+        expected-sha256: fba78a37dc3535e4686c7f57407b97d03c676e3a57beac5fb2315162b0cc3176
+        uri: https://github.com/itstool/itstool/archive/${{package.version}}/itstool-${{package.version}}.tar.gz
+    - runs: |
+        # The tests expect to run `itstool_test` in the root of the repo
+        ln -s /usr/bin/itstool itstool_test
+
+        cd tests
+        python3 -m unittest run_tests.py

--- a/itstool.yaml
+++ b/itstool.yaml
@@ -1,7 +1,7 @@
 package:
   name: itstool
   version: 2.0.7
-  epoch: 0
+  epoch: 1
   description: ITS-based XML translation tool
   copyright:
     - license: GPL-3.0-or-later
@@ -15,7 +15,7 @@ environment:
       - busybox
       - ca-certificates-bundle
       - libxml2-py3
-      - python3
+      - python-3.12
 
 pipeline:
   - uses: fetch
@@ -26,7 +26,7 @@ pipeline:
   - runs: autoreconf -vif
 
   - runs: |
-      PYTHON="/usr/bin/python3" ./configure \
+      PYTHON="/usr/bin/python3.12" ./configure \
         --host=${{host.triplet.gnu}} \
         --build=${{host.triplet.gnu}} \
         --prefix=/usr \

--- a/itstool.yaml
+++ b/itstool.yaml
@@ -5,6 +5,9 @@ package:
   description: ITS-based XML translation tool
   copyright:
     - license: GPL-3.0-or-later
+  dependencies:
+    runtime:
+      - libxml2-py3
 
 environment:
   contents:

